backend/control_center/.env.example

```
@@ -50,4 +50,4 @@ TELEGRAM_API_KEY=your-telegram-api-key-here
 # Set this to the published Docker Hub image you want to use
 # Format: username/repository:tag or repository:tag
 # Example: myusername/sdn-launch-control-backend:latest
-# DOCKER_IMAGE=sdn-launch-control-backend:latest
\ No newline at end of file
+DOCKER_IMAGE=taurinetech/sdn-launch-control-backend:1.0.0-beta
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)